pyproject.toml

```diff
@@ -28,34 +28,45 @@ python = ">=3.8,<4.0"
 pip = "^23.0.1"
```

Comment:
python 3.8 won't work because `dict[str, dict[str, str]]` syntax is used. 
we need either to make syntax compatible with python 3.8 or edit requirement to python >=3.9

Reply:
removed support for python 3.8